OCTRI/poc-hpo-on-fhir
# Upgrade/Add ability to report phenotype discoveries back to FHIR server

- Upgrade to latest fhir2hpo and hapi-fhir libraries
- Create an R5 FHIR Service and focus on connecting only to R5 for Connectathon
- Add actions for reporting the phenotypes back to the FHIR service with comments
- Check for phenotypes already reported and display a checkbox when no update is needed

Feel free to ignore most of this pull request, but some feedback on phenotypes.html and hposummary.js would be helpful.
